package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "di-ipv-cri-common-express",
-  "version": "0.0.29",
+  "version": "0.0.30",
```

Comment:
Don't update the version number in a PR - make a separate one with just this update, and then release via Github.

Otherwise, this can cause other PRs to accidentally be included in a release.